### PR TITLE
[Invoices] Display the shipping category.

### LINKED ISSUE #11760: [Invoices] Change the display of the shipping method name
## Description

**- As a:** enterprise user with invoices toggle ON
**- On page:** on PDF invoices
**- I want to be able to do:** (

currently in the "item" column we display 

**Shipping** (Type: `Shipping method name`). the proposal is to display: 

`Shipping method category`: `Shipping method name`

the shipping method category is defined by the user when creating a shipping method and can be Delivery or Pick up 

image

## Acceptance Criteria & Tests

1. Turn on invoices toggle
2. Check the display of the shipping method on the invoice for both pickup and delivery

### PR DESCRIPTION
**Pull request by abdellani**
#### What? Why?

- Closes #11760

#### What should we test?

- create an order.
- generate at least one invoice
- check if you can see the right shipping category

#### Release notes

Changelog Category (reviewers may add a label for the release notes):

- [x] User facing changes
- [ ] API changes (V0, V1, DFC or Webhook)
- [ ] Technical changes only
- [ ] Feature toggled

The title of the pull request will be included in the release notes.

### CODE REVIEW

File: app/views/spree/admin/orders/_invoice_table4.html.haml [hunk 1]
```diff
@@ -37,8 +37,8 @@
           = item.display_amount_with_adjustments_and_with_taxes
     %tr
       %td
-        %strong= "#{t(:shipping)} "
-        = "( #{t(:invoice_shipping_type)} #{raw(@order.shipping_method.name)} )"
+        %strong= "#{@order.shipping_method.category} : "
+        = "(#{raw(@order.shipping_method.name)})"
```
Comment (mkllnk) [inline app/views/spree/admin/orders/_invoice_table4.html.haml:41]: Does this have to be raw? I don't want enterprises to inject code via the shipping method name.

**Review by mkllnk — APPROVED** [review]

File: app/views/spree/admin/orders/_invoice_table4.html.haml (hunk 1 above)
Comment (rioug) [reply app/views/spree/admin/orders/_invoice_table4.html.haml:41]: No need for `raw` here.

**Review by rioug — CHANGES_REQUESTED** [review]
Looks good, could we just remove the `raw` in the template ?

**Review by rioug — APPROVED** [review]

**Comment by RachL** [comment]
@abdellani this is perfect. A tiny comment: can we either remove the ":" or remove the brackets? It's a bit weird to have both, but otherwise all good, ready to merge here 💪 

image

**Comment by abdellani** [comment]
ok @RachL 
I'll remove the column ':' :)

**Comment by abdellani** [comment]
@RachL  done

**Comment by RachL** [comment]
@abdellani thanks! It looks like we need to rebase right?

**Pull request merged**